internal/client/consensus/grandpa/grandpa.go

```diff
@@ -61,3 +61,96 @@ func (c Config) name() string {
 	}
 	return *c.Name
 }
+
+// Errors that can occur while voting in GRANDPA.
+var (
+	// ErrClient means we could not complete a round on disk.
+	ErrClient = errors.New("could not complete a round on disk")
```

Comment:
What does completing a round on disk mean?

Reply:
This error is emitted in the process of eventually trying to store the justification of the best block to disk.  The error is emitted from `NewGrandpaJustificationFromCommit`.